pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -51,83 +51,77 @@ type NamespaceController struct {
 	getData func() map[string]string
 	client  corev1.CoreV1Interface
 
-	queue queue.Instance
-
 	// Controller and store for namespace objects
 	namespaceController cache.Controller
-	// Controller and store for ConfigMap objects
-	configMapController cache.Controller
```

Comment:
why is this removed? What happens if the config map is removed/changed?

Reply:
This is removed to simplify the implementation. If the config map is removed/changed, the namespace controller will periodically write the configmap again.